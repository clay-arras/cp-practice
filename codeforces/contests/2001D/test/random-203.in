1
17
15 1 1 22 8 12 1 4 19 15 24 6 13 4 17 1 21
