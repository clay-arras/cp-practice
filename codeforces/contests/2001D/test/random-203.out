12
22 8 12 1 19 15 24 6 13 4 17 21
